asmcomp/amd64/emit.mlp

```diff
@@ -786,8 +786,11 @@ let emit_instr fallthrough i =
   | Lop(Ispecific(Ibswap _)) ->
       assert false
   | Lop(Ispecific Isqrtf) ->
+      if arg i 0 <> res i 0 then
+        I.xorpd (res i 0) (res i 0);
       I.sqrtsd (arg i 0) (res i 0)
   | Lop(Ispecific(Ifloatsqrtf addr)) ->
+      I.xorpd (res i 0) (res i 0);
```

Comment:
Perhaps a comment is in order?  `(* avoid partial register stall *)`

Reply:
Fixed